.github/workflows/aks-db-backup.yml

```diff
@@ -5,22 +5,43 @@ on:
   schedule: # 01:00 UTC
     - cron: "0 1 * * *"
 
+env:
+  SERVICE_NAME: faltrn
+  SERVICE_SHORT: faltrn
+  TF_VARS_PATH: terraform/aks/workspace_variables
+
 jobs:
   backup:
     name: Backup AKS Database
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      id-token: write
     strategy:
       max-parallel: 1
       matrix:
         environment: [development, test, preproduction, production]
     environment:
       name: ${{matrix.environment}}
+    env:
+      DEPLOY_ENV: ${{ inputs.environment || 'production'  }}
+      BACKUP_FILE: ${{ inputs.backup-file || 'schedule'  }}
     concurrency: ${{matrix.environment}}_${{github.event.number}}
     steps:
       - name: Check out the repo
         uses: actions/checkout@v4
+
+      - name: Set environment variables
+        run: |
+          source global_config/${DEPLOY_ENV}.sh
+          tf_vars_file=${TF_VARS_PATH}/${DEPLOY_ENV}.tfvars.json
+          echo "NAMESPACE=$(jq -r '.namespace' ${tf_vars_file})" >> $GITHUB_ENV
+          echo "CLUSTER=$(jq -r '.cluster' ${tf_vars_file})" >> $GITHUB_ENV
+
```

Comment:
I don't think L36-39 are required? We don't use the 2 vars here or pass them anywhere. They are set in the database-action based on the env